service/src/main/java/com/ge/predix/acs/attribute/cache/AbstractAttributeCache.java

```diff
@@ -0,0 +1,62 @@
+package com.ge.predix.acs.attribute.cache;
+
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.ge.predix.acs.model.Attribute;
+import com.ge.predix.acs.policy.evaluation.cache.AbstractPolicyEvaluationCache;
+
+@Component
+public abstract class AbstractAttributeCache implements AttributeCache {
```

Comment:
Discussed this with @anubhavi25 .  Added notes in redis caching story. 
* this pull request will be limited to in-memory caching only
* initially a common cache across zones is ok, however we need to enforce a max storage (at least max entries) on it
* key should be zone+id.   One cache each for attribute and subject connectors.